### DISCUSSION TITLE
【求助】大三学生该去考研还是当盗洞客？

### DISCUSSION THREAD

**Discussion by zhangyj0523** [share121/inter-knot#535] [其他]
![403952267104860](https://github.com/user-attachments/assets/c20d5040-f777-4a73-8a5a-0be1a7a0ea7a)
听说有从空洞里偷出千万艺术品的故事，还能体验刺激的生活，好羡慕啊。
我开学大四，考研还没开始学，以太适应性中等偏上。感觉就业好难，有没有前辈指导一下，我该去考研还是当盗洞客等？
如果说考研有机会的话，我从现在就开始学！！！
求各位绳匠，委托人大佬指点！！！

**Comment by lllliufeng** [comment]
太适应性中上...可以当调查员了吧，慕了

**Comment by MiaoJiSir** [comment]
【盗洞客的诞生】 完成任务后到喵吉长官处领取奖励！

**Comment by SoraGinko114** [comment]
天生调查员体质，羡慕了

**Comment by wujiuli39** [comment]
刺激人生，尝试吧！

**Comment by 117518** [comment]
建议先加入治安官，等拥有一定战斗能力之后，再去尝试当盗洞客

**Comment by mhpmc** [comment]
可以尝试下申请调查员

**Comment by MissHedwig** [comment]
为什么不考公？（）

**Comment by Tragedysaver1024** [comment]
楼主你好，当盗洞客违法，我的建议是请您继续考研，二战三战都考不了之后可以试试去当盗洞客。

**Comment by zhangyj0523** [comment]
谢谢大佬们，决定先考研了。家人朋友们都反对我去空洞里冒险，年轻人果然还是不要太年轻。。。

**Comment by FFT66** [comment]
【治安局提醒您】遵纪守法！
